inst/extdata/PA2022CH_fr_exec_summary/scorecard.Rmd

```diff
@@ -554,11 +554,11 @@ _Les informations pour cet indicateur n'ont pas été collectées dans le test c
-  * Mise en œuvre: «OUI (type d'actif)» signifie que l'utilisateur a investi dans le type d'actif ET a indiqué qu’il exerce des droits de vote. «NON» signifie que l'utilisateur a investi dans le type d'actif ET N’A PAS indiqué qu’il exerce des droits de vote. «PAS DE RÉPONSE» : signifie que l'utilisateur n'a pas indiqué qu'il possède des investissements dans le type d'actif. Pairs: Nombre d'utilisateurs ayant répondu OUI pour le type d'actif concerné sur l’ensemble des utilisateurs ayant investi dans ce type d'actif. Les types d'actifs couverts sont "actions cotées (LE)", "capitaux privées (PE)", "infrastructures (INF)", et "Autres actifs".
+  * Mise en œuvre: \guillemetleft OUI (type d'actif)\guillemetright signifie que l'utilisateur a investi dans le type d'actif ET a indiqué qu’il exerce des droits de vote. \guillemetleft NON\guillemetright signifie que l'utilisateur a investi dans le type d'actif ET N’A PAS indiqué qu’il exerce des droits de vote. \guillemetleft PAS DE RÉPONSE\guillemetright : signifie que l'utilisateur n'a pas indiqué qu'il possède des investissements dans le type d'actif. Pairs: Nombre d'utilisateurs ayant répondu OUI pour le type d'actif concerné sur l’ensemble des utilisateurs ayant investi dans ce type d'actif. Les types d'actifs couverts sont "actions cotées (LE)", "capitaux privées (PE)", "infrastructures (INF)", et "Autres actifs".
 * Les stratégies d'engagement doivent être considérées comme des mesures pertinentes pour le climat
-  * Mise en œuvre : conditions pour «OUI (type d'actif)», «NON», «PAS DE RÉPONSE» définies comme ci-dessus mais pour des stratégies d'engagement. Pour les « autres actifs », les conditions ci-dessus doivent s'appliquer à au moins un des autres actifs. Pairs: Nombre d'utilisateurs ayant répondu OUI pour le type d'actif concerné sur l’ensemble des utilisateurs ayant investi dans ce type d'actif. Nombre d'utilisateurs ayant répondu OUI pour d'autres actifs sur le nombre d'utilisateurs ayant investi dans au moins un autre type d'actif. Les types d'actifs couverts sont "actions cotées (LE)", "obligations d'entreprises (CB)", "immobiliers (RE)", et "Autres actifs".
+  * Mise en œuvre : conditions pour \guillemetleft OUI (type d'actif)\guillemetright , \guillemetleft NON\guillemetright , \guillemetleft PAS DE RÉPONSE\guillemetright définies comme ci-dessus mais pour des stratégies d'engagement. Pour les \guillemetleft autres actifs\guillemetright , les conditions ci-dessus doivent s'appliquer à au moins un des autres actifs. Pairs: Nombre d'utilisateurs ayant répondu OUI pour le type d'actif concerné sur l’ensemble des utilisateurs ayant investi dans ce type d'actif. Nombre d'utilisateurs ayant répondu OUI pour d'autres actifs sur le nombre d'utilisateurs ayant investi dans au moins un autre type d'actif. Les types d'actifs couverts sont "actions cotées (LE)", "obligations d'entreprises (CB)", "immobiliers (RE)", et "Autres actifs".
```

Comment:
is there supposed to be a space between open bracket and the text `\guillemetleft autres actifs\guillemetright`

Reply:
no, but you mention a valid point, that I did not menage to fix yet. `\guillemetright` seems to automitically remove a space that follows. that means it is hard to use this well within a sentence. No problem if followed by a comma or a period. I have not figured out how to force a space in latex, which we would have to do in some cases